core/models.py

```diff
@@ -1,3 +1,31 @@
 from django.db import models
+from django.utils.datetime_safe import datetime
+from django.template.defaultfilters import slugify
+from django.utils.translation import ugettext as _
 
-# Create your models here.
+
+# Post model
+class Post(models.Model):
+    title = models.CharField(max_length=255, null=False, blank=False, verbose_name=_('Заголовок'))
+    body = models.TextField(null=False, blank=False, verbose_name=_('Тест поста'))
+    created_at = models.DateTimeField(null=False, blank=False, verbose_name=_('Создано в'))
+    published_at = models.DateTimeField(null=True, blank=False, verbose_name=_('Опубликовано в'))
```

Comment:
what is the reason to keep two fields: `created_at` and `publisted_at`?  Do you **really** want to know the exact time when the post was created as a draft and when it became publicly available?